cmd/fetchpack.go

```diff
@@ -27,7 +27,7 @@ func FetchPack(c *git.Client, args []string) error {
 	flags.BoolVar(&opts.Verbose, "verbose", false, "Be more verbose")
 	flags.Parse(args)
 	args = flags.Args()
-	opts.Depth = int32(*depth)
+	opts.Depth = int(*depth)
```

Comment:
I don't think we do anything with this, maybe we should document that's it's only parsed and not implemented?